### PR TITLE
Add contributors

### PR DESCRIPTION
**Pull request by KatherineMichel**
Changes proposed in this PR:

- Add contributors

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -466,12 +467,216 @@
...
+    {
+      "login": "noahalorwu",
+      "name": "Noah Alorwu",
+      "avatar_url": "https://avatars.githubusercontent.com/u/7665391?v=4",
+      "profile": "https://github.com/noahalorwu",
+      "contributions": []
+    },
+    {
+      "login": "jalakoo",
+      "name": "jalakoo",
```
Comment (drewbrew) [inline .all-contributorsrc]:
```suggestion
      "name": "Jason Koo",
```

File: .all-contributorsrc [hunk 2]
```diff
@@ -466,12 +467,216 @@
...
+    {
+      "login": "epicserve",
+      "name": "Brent O'Connor",
+      "avatar_url": "https://avatars.githubusercontent.com/u/191620?v=4",
+      "profile": "http://www.epicserve.com/",
+      "contributions": []
+    },
+    {
+      "login": "lmann4",
+      "name": "lmann4",
```
Comment (drewbrew) [inline .all-contributorsrc]:
```suggestion
      "name": "Levi Mann",
```

File: .all-contributorsrc [hunk 3]
```diff
@@ -466,12 +467,216 @@
...
+    {
+      "login": "simonw",
+      "name": "Simon Willison",
+      "avatar_url": "https://avatars.githubusercontent.com/u/9599?v=4",
+      "profile": "https://simonwillison.net/",
+      "contributions": []
+    },
+    {
+      "login": "ltagliaferri",
+      "name": "ltagliaferri",
```
Comment (drewbrew) [inline .all-contributorsrc]:
```suggestion
      "name": "Lisa Tagliaferri",
```

**Review by drewbrew — APPROVED** [review]
I added suggestions for the names of speakers I recognized. Feel free to ignore them.

Thanks!

**Comment by KatherineMichel** [comment]
Good idea! I fixed those names. I will merge now.

**Pull request merged**